start.R

```diff
@@ -213,6 +213,14 @@ if (any(c("--testOneRegi", "--debug", "--quick") %in% flags) & "--restart" %in%
   if (gms::getLine() %in% c("Y", "y")) flags <- c(flags, "--reprepare")
 }
 
+# Check if dependencies for a model run are fulfilled
+if (packageVersion("lucode2") >= "0.31.0") {
```

Comment:
```suggestion
if (packageVersion("lucode2") >= "0.33.0") {
```
If we need https://github.com/pik-piam/lucode2/pull/126 we should require lucode2 0.33.0